it/controllers/employment/EmployerLeaveDateControllerISpec.scala

```diff
@@ -0,0 +1,927 @@
+    val futureDateError = "The date your client left their employment must be in the past"
+    val tooLongAgoDateError = s"The date your client left their employment must be after 5 April ${taxYearEOY-1}"
+    val beforeStartDateError = s"The date your client left their employment cannot be before 1 January 2021"
+  }
+
+  object ExpectedAgentCY extends SpecificExpectedResults {
+    val expectedTitle = "When did your client leave their employer?"
+    val expectedH1 = s"When did your client leave $employerName?"
+    val expectedErrorTitle = s"Error: $expectedTitle"
+    val emptyDayError = "The date your client left their employment must include a day"
+    val emptyMonthError = "The date your client left their employment must include a month"
+    val emptyYearError = "The date your client left their employment must include a year"
+    val emptyDayYearError = "The date your client left their employment must include a day and year"
+    val emptyMonthYearError = "The date your client left their employment must include a month and year"
+    val emptyDayMonthError = "The date your client left their employment must include a day and month"
+    val emptyAllError = "Enter the date your client left their employment"
+    val invalidDateError = "The date your client left their employment must be a real date"
+    val tooRecentDateError = s"The date your client left their employment must be before 6 April $taxYearEOY"
+    val futureDateError = "The date your client left their employment must be in the past"
+    val tooLongAgoDateError = s"The date your client left their employment must be after 5 April ${taxYearEOY-1}"
+    val beforeStartDateError = s"The date your client left their employment cannot be before 1 January 2021"
+  }
+
+  object CommonExpectedEN extends CommonExpectedResults {
+    val expectedCaption: Int => String = (taxYear: Int) => s"Employment for 6 April ${taxYear - 1} to 5 April $taxYear"
+    val expectedButtonText = "Continue"
+    val day = "day"
+    val month = "month"
+    val year = "year"
+    val forExample = "For example, 12 11 2007"
+  }
+
+  object CommonExpectedCY extends CommonExpectedResults {
+    val expectedCaption: Int => String = (taxYear: Int) => s"Employment for 6 April ${taxYear - 1} to 5 April $taxYear"
+    val expectedButtonText = "Continue"
+    val day = "day"
+    val month = "month"
+    val year = "year"
+    val forExample = "For example, 12 11 2007"
+  }
+
+  val userScenarios: Seq[UserScenario[CommonExpectedResults, SpecificExpectedResults]] = {
+    Seq(UserScenario(isWelsh = false, isAgent = false, CommonExpectedEN, Some(ExpectedIndividualEN)),
+      UserScenario(isWelsh = false, isAgent = true, CommonExpectedEN, Some(ExpectedAgentEN)),
+      UserScenario(isWelsh = true, isAgent = false, CommonExpectedCY, Some(ExpectedIndividualCY)),
+      UserScenario(isWelsh = true, isAgent = true, CommonExpectedCY, Some(ExpectedAgentCY)))
+  }
+
+  object CyaModel {
```

Comment:
CyaModel object isn't used - can be removed

Reply:
removed